src/backend/tests/projects.test.ts

```diff
@@ -917,19 +917,13 @@ describe('Projects', () => {
       );
```

Comment:
In the notfound case you use NERSoftware Tools which does exist. instead use like 'fakeId' because this should not be return a not found exception